pages/common/snowsql.md

```diff
@@ -0,0 +1,24 @@
+# snowsql
+
+> SnowSQL command-line client for Snowflake's Data Cloud.
+> More information: <https://docs.snowflake.com/en/user-guide/snowsql.html>.
+
+- Connect to a specific instance at <https://account.snowflakecomputing.com> (password can be provided in prompt or configuration file):
+
+`snowsql --accountname {{account}} --username {{username}} --dbname {{database}} --schemaname {{schema}}`
+
+- Connect to an instance specified by a specific configuration file (defaults to `~/.snowsql/config`):
+
+`snowsql --config {{path/to/configuration_file}}`
+
+- Connect to the default instance, specifying a MFA token:
```

Comment:
Not sure about `MFA`, I think readers will Google for `MFA` :thinking: . What if ...?
```suggestion
- Connect to the default instance using a token for multi-factor authentication:
```
I based this suggestion using https://docs.snowflake.com/en/user-guide/snowsql-start.html#connection-syntax.

 @kbdharun wdyt?

Reply:
@Holek ?

Reply:
> Not sure about `MFA`, I think readers will Google for `MFA` :thinking: . What if ...?
> ```suggestion
> - Connect to the default instance using a token for multi-factor authentication:
> ```
> I based this suggestion using https://docs.snowflake.com/en/user-guide/snowsql-start.html#connection-syntax.
> 
>  @kbdharun wdyt?

Agreed, It would be nice to specify it for new users.

Reply:
Sure, what's funny is that I explained that as such in Polish translation already.